app/src/components/Recipe.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {ReactElement, ReactNode} from 'react';
+import {IngredientType, ShowRecipeType} from "../interfaces";
+import Ingredient from "./Ingredient";
+import axios from "axios";
+import {Badge, Button, Card, Flex, FormattedText, useToast} from "@travelperksl/suitcase";
+import styled from "styled-components";
+import {Link} from "react-router-dom";
+import {actions, errors, responses} from "../lang";
+
+const Recipe = ({recipe}: { recipe: ShowRecipeType }): ReactElement => {
+    const addToast = useToast();
+
+    const handleDelete = (id: string): void => {
+        axios.delete(`${process.env.REACT_APP_BACKEND_API_HOST}/recipes/${id}`)
```

Comment:
We use [react-query](https://www.notion.so/travelperk/Using-react-query-in-TK-8e90a107c60b4321aba085f9048ef60c?pvs=4) for making http requests (I think from some brief reading react-query uses axios under the hood)